### ISSUE TITLE
More fields for jetsmart search & jetengine

### ISSUE THREAD

**Issue by neosaveme-zz** [CrocoBlock/suggestions#1835]
Hello,
I would like to leave a message about jetsmart search and jet engine,

to begin , jetsmart search approach to perfection. but the solution is very far from the flexibility compared to the facetwp solution which I was forced to use especially for the range number fields.

I hope that you will develop the potential of your solution and give us the possibility of building a true real estate type search sites.

for jetengine I like the solution but the need to create fields like ACF do. (for example taxonomy fields useful for creating a single clean interface) gives us the feeling of being limited.

today I think the potential of your solutions is penalized by this lack of flexibility.
we have to install lots of plugins and waste hours to fill this gap

to summarize you have two solutions

ACF pro and Facetwp to copy what they do best
